android_app/app/src/main/res/xml/graph_preferences.xml

```diff
@@ -24,10 +25,41 @@
         android:summaryOff="@string/info_is_not_visible"
         android:summaryOn="@string/info_is_visible"
         android:title="@string/label_goal_line" />
-    <CheckBoxPreference
-        android:defaultValue="false"
-        android:key="trendLine"
-        android:summaryOff="@string/info_is_not_enable"
-        android:summaryOn="@string/info_is_enable"
-        android:title="@string/label_trend_line" />
+    <PreferenceCategory
+        app:key="trendlineCategory"
+        app:title="@string/label_trend_line">
+
+        <CheckBoxPreference
+            android:defaultValue="false"
+            android:key="trendLine"
+            android:summaryOff="@string/info_is_not_enable"
+            android:summaryOn="@string/info_is_enable"
+            android:title="@string/label_trend_line" />
+        <DropDownPreference
+            android:id="@+id/trendlineComputationMethodDropdown"
+            android:defaultValue="Exponentially Smoothed Moving Average"
+            android:key="trendlineComputationMethod"
+            android:dependency="trendLine"
+            android:entries="@array/trendline_computation_methods_entries"
+            android:entryValues="@array/trendline_computation_methods_values"
+            app:useSimpleSummaryProvider="true"
+            android:title="@string/label_trendline_computation_method"
+            />
+        <SeekBarPreference
+            android:id="@+id/simpleMovingAverageNumDaysSeekbar"
+            android:defaultValue="7"
+            android:key="simpleMovingAverageNumDays"
+            android:dependency="trendLine"
+            android:max="30"
+            app:min="2"
+            app:showSeekBarValue="true"
+            android:title="@string/label_simple_moving_average_num_days" />
+        <CheckBoxPreference
```

Comment:
maybe its better to move that under the trendline category?

Reply:
I moved it, but there is now a small issue (at least on my emulator) that when the user selects Simple Moving Average, the seeker bar is below the screen, so the user doesn't know it appears.

Reply:
thanks fixed that.